# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/android/AndroidContentController.cpp
# Commit: 76279f0a6d33
# Full Hash: 76279f0a6d33d6fe128a9f3aabe5393e52181818
# Author: James Willcox <snorp@snorp.net>
# Date: 2020-06-08 21:38:11
# Regressor Bug: 1536833
# File Overlap Count: 1
# Description:
#   Bug 1536833 - Fix overscroll effect on Android r=botond,geckoview-reviewers,agi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78130
# ==============================================================================

diff -r ad757ab87752 -r 76279f0a6d33 widget/android/AndroidContentController.cpp
--- a/widget/android/AndroidContentController.cpp	Sun Jun 07 23:15:45 2020 +0000
+++ b/widget/android/AndroidContentController.cpp	Mon Jun 08 17:13:28 2020 +0000
@@ -24,14 +24,16 @@
 }
 
 void AndroidContentController::UpdateOverscrollVelocity(
-    const float aX, const float aY, const bool aIsRootContent) {
+    const ScrollableLayerGuid& aGuid, const float aX, const float aY,
+    const bool aIsRootContent) {
   if (aIsRootContent && mAndroidWindow) {
     mAndroidWindow->UpdateOverscrollVelocity(aX, aY);
   }
 }
 
 void AndroidContentController::UpdateOverscrollOffset(
-    const float aX, const float aY, const bool aIsRootContent) {
+    const ScrollableLayerGuid& aGuid, const float aX, const float aY,
+    const bool aIsRootContent) {
   if (aIsRootContent && mAndroidWindow) {
     mAndroidWindow->UpdateOverscrollOffset(aX, aY);
   }